Задание 1.
Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

In [50]:
sigma = 16
alfa = 0.05
M = 80
n = 256

Известна сигма. По Z таблице определяем Z от alfa/2 (т.е. Z для 97.5%).

In [51]:
Z = 1.96

In [52]:
M + Z * (sigma/n**0.5)

81.96

In [53]:
M - Z * (sigma/n**0.5)

78.04

Ответ: [78.04:81.96]

Задание 2.
В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [54]:
import numpy as np

In [55]:
arr = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
alfa = 0.05
n = 10

In [56]:
M = round(arr.mean(), 2)
M

6.59

Сигма неизвестна, находим критерий Стьюдента t от alfa/2 (т.е. t для 97.5%). 

In [64]:
t = 2.262

Найдем несмещенную сигму.

In [65]:
sigma_n = round(arr.std(ddof=1), 2)
sigma_n

0.45

In [66]:
round(M + t * (sigma_n/n**0.5), 2)


6.91

In [67]:
round(M - t * (sigma_n/n**0.5), 2)


6.27

Ответ: [6.27:6.91]

Задание 3.
Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [69]:
M = 17
X = 17.5
alfa = 0.05
n = 100
D = 4

X > M, распределение H1 справа от H0. Есть дисперсия, значит есть сигма. По Z таблице определяем границу альфа = 5%.

In [70]:
Z0 = 1.645

Находим расчетное Z.

In [71]:
sigma = D**0.5
sigma

2.0

In [72]:
Z1 = (X - M) / (sigma / n**0.5)
Z1

2.5

Ответ. Z1 > Z2, гипотеза H0 неверная на уровне значимости 5%.

Задание 4.
Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [74]:
M = 200
n = 10
arr = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
alfa = 0.01

Найдем параметры для гипотезы H1.

In [77]:
X = arr.mean()
X

198.5

In [82]:
sigma_n = round(arr.std(ddof=1), 2)
sigma_n

4.45

M != X, распределение двустороннее.

Найдем критерий Стьюдента t0 табличное для alfa/2, т.к. у нас нет сигмы генеральной савокупности.

In [87]:
t0 = 3.25

Найдем расчетное t1.

In [88]:
t1 = round((X - M) / (sigma_n / n**0.5), 2)
t1

-1.07

Ответ: t1 лежит в пределах t0 и -t0, т.е. внутри H0. Гипотеза H0 верна.